dewy/collection/router.py

```diff
@@ -83,3 +83,22 @@ async def get_collection(name: PathCollection, conn: PgConnectionDep) -> Collect
         )
 
     return Collection.model_validate(dict(result))
+
+@router.delete("/{name}")
+async def delete_collection(pg_pool: PgPoolDep, name: PathCollection) -> Collection:
+    """Delete a collection and all documents contained within it."""
+    async with pg_pool.acquire() as conn:
+        id = await conn.execute(
```

Comment:
I don't think `execute` [1] is what you want -- it returns the status of the command as a string (for instance, this will likely return `DELETE 0` if nothing is deleted or `DELETE 1` if one thing is deleted.

I think you want `conn.fetchval` [2] which returns the value from the first row or maybe `fetch` or `fetchrow`.

[1] https://magicstack.github.io/asyncpg/current/api/index.html#asyncpg.connection.Connection.execute
[2] https://magicstack.github.io/asyncpg/current/api/index.html#asyncpg.connection.Connection.fetchval